internal/thumb/vips.go

```diff
@@ -76,6 +77,33 @@ func Vips(imageName string, imageBuffer []byte, hash, thumbPath string, width, h
 		size = vips.SizeBoth
 	}
 
+	iiFull := img.GetString("exif-ifd4-InteroperabilityIndex")
```

Comment:
Does this "scan" the whole file for the string, or is it a cached lookup, so that no significant slowdown is expected when generating thumbnails?

Reply:
Once that's clarified, I'll be happy to merge this PR :+1:

Reply:
Poking at vips, it looks to be a hash table lookup: https://github.com/libvips/libvips/blob/master/libvips/iofuncs/header.c#L1374-L1375

The foreach above is going over vips' own hardcoded meta items: https://github.com/libvips/libvips/blob/97be5689fee8ac4a672bdd3be3a689ad5592e862/libvips/iofuncs/header.c#L104

So I think ok?